grib/src/main/java/ucar/nc2/grib/collection/GribIosp.java

```diff
@@ -787,11 +787,26 @@ private Array makeLazyTime2Darray(Variable coord, Time2Dinfo info) {
         break;
 
       case intvU:
```

Comment:
For my understanding, what does this mean again? interval...  unique? 😅

Reply:
I think so. I'm just not quite sure I understand what "unique" means.

Also, the test dataset has 1D time coordinates but methods and such in code include 2D ,e.g., "[makeLazyTime2Darray(...)](https://github.com/Unidata/netcdf-java/blob/2f872bd32b33a662a66bcb00707239b28632be02/grib/src/main/java/ucar/nc2/grib/collection/GribIosp.java#L737)" and "enum GribIosp.[Time2DinfoType](https://github.com/Unidata/netcdf-java/blob/2f872bd32b33a662a66bcb00707239b28632be02/grib/src/main/java/ucar/nc2/grib/collection/GribIosp.java#L568)" (which includes an "is1Dtime")?

Reply:
This same function exists in another class, the `GribIospBuilder` [here](https://github.com/Unidata/netcdf-java/blob/2f872bd32b33a662a66bcb00707239b28632be02/grib/src/main/java/ucar/nc2/grib/collection/GribIospBuilder.java#L537). It appears to be unused, so I don't think you need to update it. I will look into if I can remove it.

Reply:
Yup, I saw that too. Didn't poke around enough to figure out if it was ever used. Looks like there is code in GribIosp that does instantiate a GribIospBuilder and then work with it but I haven't dug into it to see what cases might actually use the builder.

I did look at netcdf-java-ng and lots of those methods are removed from GribIosp and remain in GribIospBuilder. So, I think 5.x got the start but not completion of the transition to builders. I was hoping netcdf-java-ng might also clarify the difference between `makeLazyTime1Darray()` and `makeLazyTime2Darray()`. Unfortunately both still have the same "// only for 2d times" comment found in 5.x.